taipy/core/task/_task_manager.py

```diff
@@ -113,4 +114,6 @@ def _submit(cls, task: Union[TaskId, Task], callbacks: Optional[List[Callable]]
         task = cls._get(task_id)
         if task is None:
             raise NonExistingTask(task_id)
-        cls._scheduler().submit_task(task, callbacks, force)
+
+        submit_id = f"SUBMISSION_{str(uuid.uuid4())}"
```

Comment:
Here you can pass None as submit_id and actually create it in the scheduler.submit_task method.

Reply:
agree! in submit_task, i will check if submit_id is provided, if nothing is provided, i will generated 1